src/components/metadata/styles/metadata.scss

```diff
@@ -165,34 +152,33 @@ $list-inline-spacing: 0.2em;
   display: flex;
   flex-grow: 1;
   align-items: center;
-}
 
-.pipeline-metadata__run-command-value--no-visible {
-  opacity: 0;
+  &--no-visible {
+    opacity: 0;
+  }
 }
 
-.pipeline-metadata__toolbox .pipeline-metadata__copy-button {
-  width: 1.2em;
-  height: 1.2em;
+.pipeline-metadata__copy-button {
+  width: auto;
+  height: auto;
```

Comment:
minor question - are we taking the width and height conditions off for the `copy-button` inside `metadata_toolbox`?

Reply:
check the output in the browser - the width/height is set on the `svg` icon element in the original component, but for some reason a smaller width/height is set on the parent button in this case, which messed up how the focus state looks without changing the actual visible width/height. I think it's a mistake:
![image](https://user-images.githubusercontent.com/1155816/99993367-79beb480-2daf-11eb-8066-d5fbf66a37e0.png)

Reply:
good spot, glad that you fixed this in this PR!